app/src/pages/inside/launchesPage/LaunchToolbar/launchToolbar.scss

```diff
@@ -2,3 +2,9 @@
   background-color: $COLOR--white-two;
   border-bottom: 1px solid $COLOR--gray-91;
 }
+
+.sticky-toolbar {
+  position: sticky;
+  top: 0;
+  z-index: 2;
```

Comment:
Why 2 not 3?

Reply:
The z-index value constants are introduced in z-index-variables.sccs. I suggest to use them instead of hard coded number.

Reply:
fixed, I've added new variable to `z-index-variables`, 2 is to be displayed above the all table elements